google/ggrc-core
# GGRC-4677 Assessment - Required Comment and Evidence popup is not shown up if select dropdown option that required evidence and comment in LCA

# Issue description

Required Comment and Evidence popup is not shown up, mandatory LCA is marked with green color, Assessment moves to In progress state.

# Steps to test the changes

1. Open any audit page with a control snapshot
2. Create Assessment template with the following LCA:
title - "Dropdown";
Type - Dropdown
Dropdown Options: "one" - comment and evidence required, mandatory

3. Autogenerate assessment based on Assessment template and control snapshot
4. Expand Assessment INfo panel and set "one" option in LCA dropdown

_Actual Result_: Required Comment and Evidence popup is not shown up, mandatory LCA is marked with green color, Assessment moves to In progress state.

_Expected Result_: Required Comment and Evidence popup should be shown up, If dropdown option of LCA requires evidence, comment, evidence plus comment

# Solution description

Get properties from `validationConfig` by `attr` function instead of by _default javascript way_.

# Sanity checklist




# PR Review checklist
